### PR TITLE
BUG missing flags; better flux norm

### PR DESCRIPTION
**Pull request by beckermr**
It appears that I messed up the flags in the recent admom change. I also found a better way to match the flux normalization. The old way threw errors sometimes for reasons I did not follow.

### CODE REVIEW

**Comment by beckermr** [comment]
This bug really sucks BTW. My apologies. :(

File: CHANGES.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
       model, be careful to use the fitted parameters rather than going through
       an intermediate gmix, which converts e1, e2 to g1, g2 and can fail in
       rare cases.
+    - Some AdmomFitter failures were mot being flagged properly.
```
Comment (esheldon) [inline CHANGES.md:20]: "mot" -> "not"

**Review by esheldon — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (beckermr) [reply CHANGES.md:20]: 🤦

Comment (beckermr) [inline CHANGES.md:20]:
```suggestion
    - Some AdmomFitter failures were not being flagged properly.
```

**Review by esheldon — APPROVED** [review]

**Pull request merged**